### DISCUSSION TITLE
Is it possible to use pg_cron instead?

### DISCUSSION THREAD

**Discussion by eqoram** [travisvn/supabase-pause-prevention#1] [General]
Hi,

Awesome project! I was just wondering if you could also use something like pg_cron (https://supabase.com/docs/guides/database/extensions/pg_cron?queryGroups=database-method&database-method=sql) instead? 
This way you wouldn't need to include an extra service for the cron job although I am not sure what the specifications for marking a project as inactive by Supabase are as I was not able to find any details about this.
Basically the question is if an external call is necessary or if you could also just do an internal call inside of the Postgres database.

**Comment by travisvn** [comment]
Hey @eqoram 

That's a great question, and I do not know the answer. 

It's possible Supabase includes the edge cases for how it considers a project "inactive" in a policy document, but even if so, I would assume it's in the company's best interest to not lay out strictly how the determination is made in totality. 

Anyway, this project takes into account a typical tech stack that involves Next.js which most will host on Vercel if they are in the situation that their projects are not active enough to warrant switching to AWS or another cloud provider. 

If you'd like to experiment and see if a Supabase database extension `pg_cron` making similar calls to the database will keep the project active, I'd very much welcome your results and go from there 🙂
